src/style.css

```diff
@@ -1 +1,75 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url('fonts/Avenir-Book.ttf');
+}
+
+.form--large,
+.form--small {
+  width: 100%;
+  margin: 20px 0;
+  position: relative;
+}
+
+.search__bar {
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  line-height: 22px;
+  border: 1px solid #e1e7ed;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)